src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -101,102 +111,95 @@ trait SharedFileSystem {
-   * Creates host execution directory, inputs path, and outputs path.  Stages any input files into the workflow-inputs
-   * directory and localizes their paths relative to the container.
+   * Creates host execution directory.
    */
   def initializeForWorkflow(descriptor: WorkflowDescriptor): Try[HostInputs] = {
     val hostExecutionDirectory = LocalBackend.hostExecutionPath(descriptor).toFile
     hostExecutionDirectory.mkdirs()
-    val hostExecutionAbsolutePath = hostExecutionDirectory.getAbsolutePath
-    Array("workflow-inputs", "workflow-outputs") foreach { Paths.get(hostExecutionAbsolutePath, _).toFile.mkdir() }
-    stageWorkflowInputs(descriptor)
+    Success(descriptor.actualInputs)
   }
 
   /**
    * Return a possibly altered copy of inputs reflecting any localization of input file paths that might have
    * been performed for this `Backend` implementation.
    */
-  def adjustInputPaths(call: Call, inputs: CallInputs): CallInputs = {
-    def containerPath(path: String): WdlFile = {
+  def adjustInputPaths(callKey: CallKey, inputs: CallInputs, workflowDescriptor: WorkflowDescriptor): CallInputs = {
+    val call = callKey.scope
+    val strategies = if (call.docker.isDefined) DockerLocalizers else Localizers
+
+    def toDockerPath(path: Path): Path = {
       // Host path would look like cromwell-executions/three-step/f00ba4/call-ps/stdout.txt
       // Container path should look like /root/f00ba4/call-ps/stdout.txt
-      val fullPath = Paths.get(path).toFile.getAbsolutePath
+      val fullPath = path.toFile.getAbsolutePath
       // Strip out everything before cromwell-executions.
       val pathUnderCromwellExecutions = fullPath.substring(fullPath.indexOf(CromwellExecutionRoot) + CromwellExecutionRoot.length)
       // Strip out the workflow name (the first component under cromwell-executions).
       val pathWithWorkflowName = Paths.get(pathUnderCromwellExecutions)
-      WdlFile(WdlFile.appendPathsWithSlashSeparators("/root", pathWithWorkflowName.subpath(1, pathWithWorkflowName.getNameCount).toString))
+      Paths.get(WdlFile.appendPathsWithSlashSeparators("/root", pathWithWorkflowName.subpath(1, pathWithWorkflowName.getNameCount).toString))
     }
 
-    // If this call is using Docker, adjust input paths, otherwise return unaltered input paths.
-    def adjustPath(nameAndValue: (String, WdlValue)): (String, WdlValue) = {
-      val (name, value) = nameAndValue
-      val adjusted = value match {
-        case WdlFile(path) => containerPath(path)
-        case WdlArray(t, values) => new WdlArray(t, values map { adjustPath(name, _)._2 })
-        case WdlMap(t, values) => new WdlMap(t, values mapValues { adjustPath(name, _)._2 })
-        case x => x
-      }
-      name -> adjusted
+    /**
+     * Transform an original input path to a path in the call directory.
+     * The new path matches the original path, it only "moves" the root to be the call directory.
+     */
+    def toCallPath(path: Path): Try[Path] = Try {
```

Comment:
I'm already kinda leery w/ our current scheme of sometimes Try and sometimes letting things bubble up. Since we decided long ago to ignore the advice of the akka overlords and not let actors crash all over the place I'd prefer to put the Try around the primordial dangerous code instead of wrapping something further up in a Try (e.g. what happens if something else decides to call that underlying function)